### DISCUSSION TITLE
Microsoft Azure does not pass verification using GitHub

### DISCUSSION THREAD

**Discussion by abelGpro** [community/community#131336] [GitHub Education] [answered]
### Hi everyone,

Hi there,

A few days ago, I activated my Github Student Developer Pack using my educational institution email .It works with almost all services except Microsoft Azure.

The main problem is that I'm unable to complete the verification with Microsoft Azure to get student benefits.
When I try to go through the Azure for Students verification by opening this link:
https://signup.azure.com/studentverification?offerType=1
and using GitHub as a verification method I see this:
"
The account you are signed is not yet verified to access offer benefits.
...
We could not confirm your status as a student.
"
I already verified my status as a student on github education.
![image](https://github.com/community/community/assets/165566209/8d7cc032-7e22-45fb-8987-3a80ef8918ba)

Can you tell me what the issue is?

**Comment by davevad93** [comment, marked as answer]
Hi @abelGpro ,

It seems to be a rather common problem among users who have access to the Student Developer Pack, for example look at this discussion #48300.

I think your best shot is try to contact support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your issue with Azure verification.

**Reply by abelGpro** [reply]
Thank you for your reply! I reached out to  Microsoft Azure and solved the issue after few email exchanges. It took 4 days.

**Reply by ABIDEGEFE** [reply]
Hi abelGpro, I faced the same challenge now, unable to verify azure student service with verified githhub student pack. Would you show me steps to solve this issue?
